gafaelfawr/templates/redis-deployment.yml

```diff
@@ -0,0 +1,39 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ template "helpers.fullname" . }}-redis
+  name: {{ template "helpers.fullname" . }}-redis
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "helpers.fullname" . }}-redis
+  template:
+    metadata:
+      labels:
+        app: {{ template "helpers.fullname" . }}-redis
+    spec:
+      containers:
+      - name: redis
+        image: redis
+        imagePullPolicy: Always
+        args:
+          - redis-server
+          - --appendonly yes
+        ports:
+          - containerPort: 6379
+        resources:
+          limits:
+            cpu: "1"
+        volumeMounts:
+          - name: redis-data
+            mountPath: "/data"
+      volumes:
+        - name: redis-data
+          {{- if .Values.redis_claim }}
+          persistentVolumeClaim:
```

Comment:
You might just want to take this block from the values - that way you could put in any mount.  But this seems quick and good.